### PR TITLE
chore: update remix packages

### PR DESCRIPTION
**Pull request by Klukies**
I've update the remix packages and, as the [release notes](https://github.com/remix-run/remix/releases/tag/remix%401.19.0) state, have updated `useRouteLoaderData` to make use of the generic type within the application.

### CODE REVIEW

**Review by kentcdodds — APPROVED** [review]
Nice! I missed that bit in the release notes. Thanks!

File: app/utils/request-info.ts [hunk 1]
```diff
@@ -1,11 +1,10 @@
-import { type SerializeFrom } from '@remix-run/node'
 import { useRouteLoaderData } from '@remix-run/react'
 import { type loader as rootLoader } from '~/root.tsx'
 
 /**
  * @returns the request info from the root loader
  */
 export function useRequestInfo() {
-	const data = useRouteLoaderData('root') as SerializeFrom<typeof rootLoader>
+	const data = useRouteLoaderData<typeof rootLoader>('root')!
```
Comment (kentcdodds) [inline app/utils/request-info.ts:8]: I strongly prefer avoiding `!`. Could we use our `invariant` util instead?

Comment (Klukies) [reply app/utils/request-info.ts]: Aha, I've been working on a project that currently isn't using `invariant` so didn't jump to it.
I've added now 😊

**Review by kentcdodds — APPROVED** [review]
Thank you!

**Pull request merged**